### PR TITLE
[`Fix`] namespace scopes

### PR DESCRIPTION
**Pull request by cschuchardt88**
# Change Log

- Changed the namespace scope from `file_scoped` to `block_scoped` for some files. 
- Also change `.editorconfig` to error on future namespace `file_scoped` files.
- Also did `dotnet format` on repo to fix any other problems.

### file_scoped
```csharp
using System;

namespace My.Namespace;

public class MyClass
{
}
```

### block_scoped
```csharp
using System;

namespace My.Namespace
{
    public class MyClass
    {
    }
}
```

## Type of change

- [ ] Optimization (the change is only an optimization)
- [x] Style (the change is only a code style for better maintenance or standard purpose)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
- [x] N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Jim8y** [comment]
emmmmm, i definately love this, but many unrelated change right? that part i dont like~~~~

**Comment by cschuchardt88** [comment]
We agreed a while back that we wouldn't use `File_Scoped` namespacing in the repo. Back when I formatted this repo with `.editorconfig`.

**Comment by shargon** [comment]
Is not better the other one? Now is the default I think

**Comment by cschuchardt88** [comment]
`block_scoped` is the default one. But there is some files that are `file_scoped`. Also `block_scoped` was not set in the `.editorconfig`, but now it is and errors on `file_scoped`; to prevent  them getting committed.

**Review by shargon — COMMENTED** [review]
I feel that we are moving from A to B and from B to A

**Comment by cschuchardt88** [comment]
I don't know what a or b is. We're just removing the file scoped namespacing. So all the code syntax is the same layout. I don't know who to blame (I guess I could look). But somebody allowed file scoped namespacing to be committed to the repo I was surprised to see how many was there.

File: src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs [hunk 1]
```diff
@@ -12,31 +12,32 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-namespace Neo.Cryptography.BLS12_381;
-
-public static class ConstantTimeUtility
+namespace Neo.Cryptography.BLS12_381
 {
-    public static bool ConstantTimeEq<T>(in T a, in T b) where T : unmanaged
+    public static class ConstantTimeUtility
```
Comment (shargon) [inline src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs:17]: Please focus in the PR changes, hard to review like this

**Review by shargon — CHANGES_REQUESTED** [review]

File: src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs (hunk 1 above)
Comment (cschuchardt88) [reply src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs:17]: Its like adding the `{` so everything is wrapped with `{` `}`

**Review by Jim8y — DISMISSED** [review]

**Review by Jim8y — DISMISSED** [review]

**Review by Jim8y — APPROVED** [review]

**Review by Jim8y — DISMISSED** [review]

File: benchmarks/Neo.VM.Benchmarks/VMTypes/Benchmarks_DeepCopy.cs [hunk 2]
```diff
@@ -12,111 +12,112 @@
...
+            for (var i = 0; i < elementsPerLevel; i++)
+            {
+                var childArray = new Array(referenceCounter);
+                rootArray.Add(childArray);
+                CreateNestedArray(childArray, depth - 1, elementsPerLevel, referenceCounter);
+            }
         }
 
-        for (var i = 0; i < elementsPerLevel; i++)
+        private static void CreateNestedTestArray(TestArray rootArray, int depth, int elementsPerLevel = 1, IReferenceCounter referenceCounter = null)
```
Comment (shargon) [inline benchmarks/Neo.VM.Benchmarks/VMTypes/Benchmarks_DeepCopy.cs:98]:
```suggestion
        private static void CreateNestedTestArray(TestArray rootArray, int depth, int elementsPerLevel = 1, IReferenceCounter? referenceCounter = null)
```

File: benchmarks/Neo.VM.Benchmarks/VMTypes/Benchmarks_Convert.cs [hunk 3]
```diff
@@ -14,97 +14,95 @@
...
 {
-    private Dictionary<StackItemType, List<StackItem>>? testItemsByType;
-
-    [GlobalSetup]
-    public void Setup()
+    public class Benchmarks_Convert
     {
-        testItemsByType = CreateTestItemsByType();
-    }
+        private Dictionary<StackItemType, List<StackItem>> testItemsByType;
```
Comment (shargon) [inline benchmarks/Neo.VM.Benchmarks/VMTypes/Benchmarks_Convert.cs:21]:
```suggestion
        private Dictionary<StackItemType, List<StackItem>>? testItemsByType;
```

File: benchmarks/Neo.VM.Benchmarks/OpCode/OpCodeBase.cs [hunk 4]
```diff
@@ -11,35 +11,36 @@
...
+    public abstract class OpCodeBase
     {
-        script = CreateScript(BenchmarkMode.SimpleOpCode);
-        multiScript = CreateScript(BenchmarkMode.OneGAS);
-        baseLineScript = CreateScript(BenchmarkMode.BaseLine);
+        [Params(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2040)]
+        public int ItemCount { get; set; } = 10;
+        protected byte[] baseLineScript;
+        protected byte[] script;
+        protected byte[] multiScript;
```
Comment (shargon) [inline benchmarks/Neo.VM.Benchmarks/OpCode/OpCodeBase.cs:22]:
```suggestion
        protected byte[] baseLineScript = Array.Empty<byte>();
        protected byte[] script = Array.Empty<byte>();
        protected byte[] multiScript = Array.Empty<byte>();
```

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
Difference checked by my side

**Review by superboyiii — APPROVED** [review]
Storage comparison on mainnet shows the same. So the code logic hasn't been changed mistakenly. It seems safe.

**Comment by NGDAdmin** [comment]
@cschuchardt88 conflict again:)

**Review by Jim8y — DISMISSED** [review]

**Comment by shargon** [comment]
> @cschuchardt88 conflict again:)

Maybe was my fault 😅

**Pull request merged**